packages/celotool/src/cmds/deploy/upgrade/verify-contracts.ts

```diff
@@ -0,0 +1,47 @@
+import * as yargs from 'yargs'
+import { InitialArgv } from '../../deploy/initial'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { getBlockscoutUrl } from 'src/lib/endpoints'
+import { execCmd } from 'src/lib/utils'
+
+export const command = 'verify-contracts'
+
+export const describe = 'verify the celo smart contracts in blockscout'
+
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .option('skipClusterSetup', {
```

Comment:
specified but not used

Reply:
So if you do not pass any `--contract` or you use the keyword `--contract all`, you will verify all the available contracts in `protocol/build/<network>/contracts/`. We can change the syntax to allow a list or do not use the keyword `all` for processing all. If you pass one contract it will try to verify only that contract (useful for debugging mostly).

Reply:
list is not required, but if `contract != all` then we need to pass the contract address to the script